homework_solution.md

```diff
@@ -0,0 +1,474 @@
+   Union Station                                 |  7001
+   Queen St W / Portland St                      |  6955
+   ```
+
+4. _**What are the most popular stations to bike from in the winter?**_
+
+   ```
+   SELECT s.name AS Station, COUNT(t.id) AS Trips
+   FROM stations s JOIN trips t
+      ON s.id = t.from_station_id
+   WHERE EXTRACT (MONTH FROM start_time) IN (12, 1, 2)
+   GROUP BY Station
+   ORDER BY Trips DESC
+   LIMIT 10;
+   ```
+
+   This yields:
+
+   ```
+                  station                 | trips
+   -----------------------------------------+-------
+   Union Station                           | 13764
+   York St / Queens Quay W                 | 12530
+   Dundas St W / Yonge St                  | 12379
+   Bay St / Wellesley St W                 | 11381
+   Bay St / College St (East Side)         | 11007
+   Queen St W / Portland St                | 10617
+   King St W / Spadina Ave                 | 10510
+   Front St W / Blue Jays Way              | 10403
+   Bay St / Queens Quay W (Ferry Terminal) | 10142
+   Ontario Place Blvd / Remembrance Dr     |  9751
+   ```
+
+5. _**Come up with a question that's interesting to you about this data that hasn't been asked and answer it.**_
+
+   We'll take a look at popular stations to start a journey from, along Yonge St.
+
+   ```
+   SELECT s.name AS Station, COUNT(t.id) AS Trips
+   FROM stations s JOIN trips_backup t
+      ON s.id = t.from_station_id
+   WHERE s.name LIKE '%Yonge%'
+   GROUP BY Station
+   ORDER BY Trips DESC
+   LIMIT 15;
+   ```
+
+   This yields:
+
+   ```
+                     station                   | trips
+   ---------------------------------------------+-------
+   Dundas St W / Yonge St                      | 36607
+   Queens Quay / Yonge St                      | 23030
+   Edward St / Yonge St                        | 18280
+   Wellesley St E / Yonge St (Green P)         | 18076
+   Front St W / Yonge St (Hockey Hall of Fame) | 17610
+   Yonge St / Wood St                          | 15248
+   Yonge St / Harbour St                       | 14003
+   Gould St / Yonge St (Ryerson University)    | 13980
+   Toronto Eaton Centre (Yonge St)             | 13263
+   Yonge St / Yorkville Ave                    | 12595
+   Yonge St / Dundonald St - SMART             |  6715
+   Yonge St / Alexander St - SMART             |  6584
+   Marlborough Ave / Yonge St                  |  3872
+   Yonge St / Aylmer Ave                       |  3739
+   Yonge St / Bloor St                         |  1230
+   ```
+
+   It's no surpise that the highest concentration is closer downtown (as can be seen on the query results plotted at https://drive.google.com/open?id=1du9Tz8PVe0Xl2FEpIi2ehkaDre4NV1ML&usp=sharing).
+
+   However, it's worthy of attention that there isn't more activity between Union and Queen St.
+   It's also interesting to see that Bloor/Yonge, being such a significant crossing and combining the 2 TTC lines, sits at 15th place.
```

Comment:
[question]
Why do you think these are the case? How could you verify that? (note: I'm not necessarily looking for a technical answer here, more contextualizing)

Reply:
For the first point, it could be that the stations are near Yonge, but not on (like King/Bay, for instance). Because we don't have geographical data for the stations, we can't really query them by distance to further analyze it.
For the second, one of the reasons could be that the station has a name excluding Yonge (like `Bloor Subway Station` or whatever, and it wouldn't come into our `Yonge` query. We'd need to dive deeper into the stations to try and find if that's hypothesis can be confirmed.